applications/osb-portal/src/components/workspace/drawer/WorkspaceResourceBrowser.tsx

```diff
@@ -18,11 +18,16 @@ import {
 } from "../../icons";
 
 
+const openFileResource = (resource: WorkspaceResource) => (e: any) => {
+  const fileName = resource.location.slice(resource.location.lastIndexOf("/")+1);
```

Comment:
This works only for resources that are on the root. Why not resource.location.slice(1) to remove the root slash and leave the rest of the workspace path?

Reply:
because the download task downloads it into the root of the given folder

Reply:
I understand, but that doesn't scale to different interpretations as `resource.location.slice(1)` does, unless I'm missing something. Resources coming from zip files files I expect not to be in the root folder for example.